docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md

```diff
@@ -194,3 +164,52 @@ DocDB uses a highly customized version of [RocksDB](http://rocksdb.org/), a log-
 | Compaction (B)             | The number of bytes being read and written to do compaction. If not a lot of data is being deleted, these levels are similar. In some cases, you might see a large delete indicated by large reads but low writes afterwards (because a large percentage of data was removed in compaction). | |
 | Compaction time (ms)           | Time (in milliseconds) for the compaction processes to complete. | |
 | Compaction num files           | The average number of files in any single compaction.        | You may consider this information while examining other metrics. |-->
+
+### DocDB
+
+[DocDB](../../../architecture/layered-architecture/#docdb) is a distributed document store, and the YugabyteDB storage layer. It uses a highly customized version of [RocksDB](http://rocksdb.org/), a log-structured merge tree (LSM)-based key-value store.
```

Comment:
Suggestion: 

[DocDB](../../../architecture/layered-architecture/#docdb) is a distributed document store that serves as the YugabyteDB storage layer. It uses a highly-customized version of [RocksDB](http://rocksdb.org/), a log-structured merge tree-based key-value store.